### DISCUSSION TITLE
Upgrading and having issue with Font color

### DISCUSSION THREAD

**Discussion by rshmhrj** [jpanther/congo#108] [Q&A] [answered]
Hi all,
I'm trying to upgrade to 2.0 and most of it is going great -- the biggest issue I'm facing is the most of the content is blending into the background:

![image](https://user-images.githubusercontent.com/750874/152749690-dfc10332-7a8f-4c88-a597-a4f3030cf9cd.png)

Unselecting these two items from the `.prose` class makes it work fine:
```css
--tw-prose-body: rgb(var(--color-neutral-700));
--tw-prose-headings: rgb(var(--color-neutral-800));
```

![image](https://user-images.githubusercontent.com/750874/152749904-256a4c16-e0af-4b62-b913-5222b7b7ee89.png)

Also, manually updating the neutral 700 and 800 to `255, 255, 255` seems to fix it.

Not sure what I did wrong... any ideas?

**Comment by jpanther** [comment, marked as answer]
Hey @rshmhrj, are you using a custom template to build this page? I notice from your screenshots that the code looks like it's different to that in the theme.

The specific issue you're having is related to the dark mode prose class. When prose is required, the correct classes are `prose dark:prose-invert`.

**Reply by rshmhrj** [reply]
You are an absolute genius!

I did have a different `single.html` created for including the disqus comments thing:

```html
    {{ if and (.Site.Params.article.showComments) (.Params.comments) }}
    <br><br>
    <section>
        {{ template "_internal/disqus.html" . }}
    </section>
    <br>
    {{ end }}
```

Copying over everything from `themes/congo/layouts/_default/single.html` to this file and re-adding this snippet in between the content and the footer is the trick:

![image](https://user-images.githubusercontent.com/750874/152900455-7ec9a3f1-5f98-4971-96d4-9693e0674cb9.png)

**Absolutely slick and gorgeous!  Great work on v2 sir!**

**Reply by jpanther** [reply]
Thanks, glad it worked. Just wondering though, is there a particular reason why you need to create your own `single.html` template for including the comments? You can provide a `layouts/partials/comments.html` file and the theme will include it for you. Doing it that way will avoid conflicts with the base template in future.
